internal/version/version.go

```diff
@@ -19,30 +19,110 @@
 package version
 
 import (
+	"encoding/json"
 	"fmt"
+	"log"
+	"runtime/debug"
+	"strings"
 
 	"github.com/spf13/cobra"
 
 	"github.com/onflow/flow-cli/build"
+	"github.com/onflow/flow-cli/internal/command"
 )
 
+type versionCmd struct {
+	Version      string
+	Commit       string
+	Dependencies []debug.Module
+}
+
+// Print prints the version information in the given format.
+func (c versionCmd) Print(format string) error {
+	switch format {
+	case command.FormatInline, command.FormatText:
+		var txtBuilder strings.Builder
+		txtBuilder.WriteString(fmt.Sprintf("Flow CLI Version: %s %s\n", c.Version, c.Commit))
```

Comment:
This will be breaking to VSCode right now... can we keep this part how it was just for the time being at least, so that I can try to get changes out that prevent this from breaking the extension first.

Reply:
oh i see, i'll swap it back to where it was before